package/yast2-bootloader.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Wed Feb 25 21:48:10 UTC 2015 - jreidinger@suse.com
+
+- Fix creation of multipath device map
```

Comment:
There is no bug, I found it when I play with tests. No product is affected, bug is introduced by one of cleaning when udev device mapping is moved to own class.